src/components/digests/DigestGuessTitle.tsx

```diff
@@ -0,0 +1,73 @@
+'use client';
+
+import message from '@/messages/en';
+import Button from '../Button';
+import { AiOutlineRobot } from '@react-icons/all-files/ai/AiOutlineRobot';
+import { useChat } from 'ai/react';
+import { useEffect, useState } from 'react';
+
+const DigestGuessTitle = ({
+  handleGuess,
+  lastDigestTitles,
+}: {
+  handleGuess: (value: string) => void;
+  lastDigestTitles: string[];
+}) => {
+  const [guessedTitle, setGuessedTitle] = useState<string>();
```

Comment:
In this case, it's more a completion than a chat : 

https://sdk.vercel.ai/docs/api-reference/use-completion

Reply:
@shinework it seems that turbo 3.5 version is a chat model only.

Reply:
Title is now guessed in digest update route when digest is published an saved in team object in base